scala/integration/intellij-bsp/src/org/jetbrains/plugins/scala/bsp/flow/open/ScalaBspProjectOpenProcessor.scala

```diff
@@ -0,0 +1,14 @@
+package org.jetbrains.plugins.scala.bsp.flow.open
+
+import com.intellij.openapi.vfs.VirtualFile
+import org.jetbrains.plugins.bsp.flow.open.BaseBspProjectOpenProcessor
+import org.jetbrains.plugins.scala.bsp.BspFeatureFlags
+import org.jetbrains.plugins.scala.bsp.config.ScalaPluginConstants
+
+
+class ScalaBspProjectOpenProcessor extends BaseBspProjectOpenProcessor(ScalaPluginConstants.BUILD_TOOL_ID) {
+  override def getName: String = "Sbt over BSP"
+
+  override def canOpenProject(projectPath: VirtualFile): Boolean =
+    BspFeatureFlags.isBspPluginIntegrationEnabled && projectPath != null && projectPath.findChild(ScalaPluginConstants.SBT_CONFIG_FILE) != null
```

Comment:
to me, the flag left here is not a good idea.
The flags are supposed to flexibly enable/disable features when you are already inside the project, but here, it is right at the opening part.
I am concerned that after you import the project using a different mechanism and turn on the flag, you will not be able to change to the `intellij-bsp` flow. 
But you can double-check that, whatever works for you.

Reply:
We have removed our key form register keys collection, so consequently we don't need isBspPluginIntegrationEnabled anymore since it's always enabled